playbooks/roles/nginx/tasks/main.yaml

```diff
@@ -0,0 +1,21 @@
+- name: Ensure Nginx is at the desired version
+  apt: pkg=nginx-full state=latest
+
+- name: Ensure main Nginx configuration file is up to date
+  template: src=nginx.conf dest=/etc/nginx/nginx.conf
+  notify: restart nginx
+
+- name: Ensure Nginx does not have a default configuration
+  file: path=/etc/nginx/sites-enabled/default state=absent
+  notify: restart nginx
+
+- name: Ensure nginx runs with the latest uwsgi configuration file
+  template: src=nginx_uwsgi.conf dest=/etc/nginx/sites-available/nginx_uwsgi.conf
+  notify:
+    - restart nginx
```

Comment:
super nit picky here, but we could keep the format similar to: 

`notify: restart nginx`

versus

```notify:
  - restart nginx
```